.github/CI/env_setup.sh

```diff
@@ -0,0 +1,31 @@
+# This file should be "sourced" into your environment
+
+# Show the executed command, but don't affect spawned shells
+trap 'echo "# $BASH_COMMAND"' DEBUG # Show commands
+
+export BUILD_TYPE=$1
+export SHARED_TYPE=$2
+export PROFILING=$3
+
+echo "Loading environment"
+source /apps/spacks/current/github_env/share/spack/setup-env.sh
```

Comment:
why an absolute path here ?

Reply:
This is where the spack env that is built for the parsec CI is located in the docker container.  It is built ahead of time to avoid issues related to trying to build the dependencies on-demand.  I don't know of a problem with using absolute paths.  There isn't a way to refer to this location without doing so.

Reply:
Such a change will prevent the parsec CI from running on any other environment than this (or any parsec forks from having their own CI). Providing an environment variable to point to the location of the installed spack could be one way of solving this. The other one would be to allow parsec to install its own spack, based on a tag/ref.

Reply:
I take back what I said yesterday about the secrets, we are not using a secret but instead we use the github slack app to send messages to our slack channel. But, I'm sure I played around with the secrets to have a different SLACK interaction, and the secrets were available in the CI script.

As a test I added an environment variable to parsec and dplasma called SPACK_SETUP_ENV. Can you see if it is available in the CI ?